tools/rum-models-generator/Sources/RUMModelsGeneratorCore/RUM/RUMSwiftTypeTransformer.swift

```diff
@@ -176,6 +186,14 @@ internal class RUMSwiftTypeTransformer: TypeTransformer<SwiftType> {
             fixedName = "RUMCITest"
         }
 
+        if fixedName == "Device" {
+            fixedName = "RUMDevice"
+        }
+
+        if fixedName == "OS" {
+            fixedName = "RUMOS"
+        }
```

Comment:
Not sure on this new name - `RUMOS` looks odd. I was considering `RUMOperatingSystem` as an alternative. Happy to hear some other ideas 🙌.

Reply:
`RUMOperatingSystem` looks better 😄

Reply:
👌🙃✅